mqtt/mqtt-bridge/tests/integration.rs

```diff
@@ -1,30 +1,24 @@
-use std::{any::Any, convert::Infallible, time::Duration, vec};
+mod common;
```

Comment:
Why declare the mod here? Isn't it usually in some lib.rs? But can we even have this under the tests folder?

Reply:
yeah, each integration test module is a separate compilation unit. so we need to declare share modules in each test module. at some point, we may want all our integration tests to be a single compilation unit but there will get different issues :)